test/integration/util/util.go

```diff
@@ -424,8 +424,7 @@ func InitTestSchedulerWithOptions(
 		t.Fatalf("Couldn't create scheduler: %v", err)
 	}
 
-	stopCh := make(chan struct{})
-	eventBroadcaster.StartRecordingToSink(stopCh)
+	eventBroadcaster.StartRecordingToSink(testCtx.Ctx.Done())
```

Comment:
https://github.com/kubernetes/kubernetes/pull/109781#issuecomment-1117272061

Please revert this part of the PR - I will take care of it once the dependent PR will merge.

[And we want to merge this PR sooner than later.]